stryker.conf.json

```diff
@@ -9,13 +8,14 @@
     "dashboard"
   ],
   "testRunner": "command",
-  "transpilers": [],
   "dashboard": {
     "project": "github.com/brodybits/xmldom",
     "version": "master"
   },
-  "coverageAnalysis": "all",
+  "coverageAnalysis": "off",
   "commandRunner": {
-    "command": "npm run test:unit"
-  }
+    "command": "npm run test:jest -- --bail --no-cache --ci --maxWorkers=1"
+  },
+  "$comment": "When all tests pass (mutant survives) our tests need some time. So this is configured to prevent 'timeout's",
```

Comment:
nit:

```
13:26:54 (18885) WARN markUnknownOptions Unknown stryker config option "$comment".
13:26:54 (18885) WARN markUnknownOptions Possible causes:
   * Is it a typo on your end?
   * Did you only write this property as a comment? If so, please postfix it with "_comment".
   * You might be missing a plugin that is supposed to use it. Stryker loaded plugins from: ["@stryker-mutator/*","/home/brodybits/xmldom/node_modules/@stryker-mutator/core/src/reporters/index.js"]
   * The plugin that is using it did not contribute explicit validation. 
   (disable "warnings.unknownOptions" to ignore this warning)
```

Reply:
Ah, nice finding, looks like I didn't pay enough attention.
So it should be `"<propName>_comment": "..."`.
Will fix that.